Broken pip install

Installing via pip seems to succeed but the installation is incomplete:
```
$ pip3 install pdal
Collecting pdal
  Downloading PDAL-2.3.6.tar.gz (942 kB)
     |████████████████████████████████| 942 kB 4.5 MB/s 
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Building wheels for collected packages: pdal
  Building wheel for pdal (PEP 517) ... done
  Created wheel for pdal: filename=PDAL-2.3.6-py3-none-any.whl size=6467 sha256=7516dac52b0808a3e075a7f91c8de5ad633cfc51c334100d9c4be4334d3f5da4
  Stored in directory: /home/ubuntu/.cache/pip/wheels/12/ef/4f/30f6a1126d2a413b49a6208034bf1262b219b848ebd3025ebd
Successfully built pdal
Installing collected packages: pdal
Successfully installed pdal-2.3.6

$ python -c "import pdal"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/ubuntu/.local/lib/python3.8/site-packages/pdal/__init__.py", line 3, in <module>
    from .pipeline import Pipeline
  File "/home/ubuntu/.local/lib/python3.8/site-packages/pdal/pipeline.py", line 2, in <module>
    from pdal import libpdalpython
ImportError: cannot import name 'libpdalpython' from partially initialized module 'pdal' (most likely due to a circular import) (/home/ubuntu/.local/lib/python3.8/site-packages/pdal/__init__.py)
```

Yes it is (built from source):
```
 pdal --version
-----------------------------------------------------------------------------------------------------------------------------------
pdal 2.2.0 (git-version: Release)
-----------------------------------------------------------------------------------------------------------------------------------
```

I managed to build pdal-python manually from source after installing scikit-build:
```
$ python setup.py build
Traceback (most recent call last):
  File "setup.py", line 22, in <module>
    from skbuild import setup
ModuleNotFoundError: No module named 'skbuild'

$ pip3 install scikit-build
Collecting scikit-build
  Downloading scikit_build-0.11.1-py2.py3-none-any.whl (72 kB)
     |████████████████████████████████| 72 kB 1.5 MB/s 
Requirement already satisfied: packaging in /usr/local/lib/python3.8/dist-packages (from scikit-build) (20.7)
Requirement already satisfied: wheel>=0.29.0 in /usr/lib/python3/dist-packages (from scikit-build) (0.34.2)
Requirement already satisfied: setuptools>=28.0.0; python_version >= "3" in /usr/lib/python3/dist-packages (from scikit-build) (45.2.0)
Requirement already satisfied: distro in /usr/lib/python3/dist-packages (from scikit-build) (1.4.0)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.8/dist-packages/pyparsing-3.0.0b1-py3.8.egg (from packaging->scikit-build) (3.0.0b1)
Installing collected packages: scikit-build
Successfully installed scikit-build-0.11.1

$ python setup.py build


--------------------------------------------------------------------------------
-- Trying "Ninja" generator
--------------------------------
---------------------------
----------------------
-----------------
------------
-------
--
Not searching for unused variables given on the command line.
-- The C compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- The CXX compiler identification is GNU 9.3.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /mnt/data/archives/python-2.3.6/_cmake_test_compile/build
--
-------
------------
-----------------
----------------------
---------------------------
--------------------------------
-- Trying "Ninja" generator - success
--------------------------------------------------------------------------------

Configuring Project
  Working directory:
    /mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-build
  Command:
    cmake /mnt/data/archives/python-2.3.6 -G Ninja -DCMAKE_INSTALL_PREFIX:PATH=/mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-install -DPYTHON_EXECUTABLE:FILEPATH=/usr/bin/python -DPYTHON_VERSION_STRING:STRING=3.8.5 -DPYTHON_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPYTHON_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libpython3.8.so -DSKBUILD:BOOL=TRUE -DCMAKE_MODULE_PATH:PATH=/usr/local/lib/python3.8/dist-packages/skbuild/resources/cmake -DCMAKE_BUILD_TYPE:STRING=Release

-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Python3: /usr/bin/python3.8 (found version "3.8.5") found components: Interpreter Development NumPy 
-- Found PythonInterp: /usr/bin/python (found version "3.8.5") 
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython3.8.so (found version "3.8.5") 
-- Found Cython: /usr/bin/cython3  
-- The project is built using scikit-build
-- Performing Test Weak Link MODULE -> SHARED (gnu_ld_ignore) - Success
_modinit_prefix:PyInit_
-- PROJECT_NAME: PDAL
-- PROJECT_NAME: PDAL
-- Configuring done
-- Generating done
-- Build files have been written to: /mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-build
[10/18] Building CXX object pdal/CMakeFiles/libpdalpython.dir/PyPipeline.cpp.o
In file included from /usr/include/python3.8/numpy/ndarraytypes.h:1830,
                 from /usr/include/python3.8/numpy/ndarrayobject.h:12,
                 from /usr/include/python3.8/numpy/arrayobject.h:4,
                 from ../../../pdal/PyPipeline.cpp:42:
/usr/include/python3.8/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
[11/18] Building CXX object pdal/CMakeFiles/libpdalpython.dir/libpdalpython.cxx.o
In file included from /usr/include/python3.8/numpy/ndarraytypes.h:1830,
                 from /usr/include/python3.8/numpy/ndarrayobject.h:12,
                 from /usr/include/python3.8/numpy/arrayobject.h:4,
                 from pdal/libpdalpython.cxx:619:
/usr/include/python3.8/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
[12/18] Building CXX object pdal/CMakeFiles/libpdalpython.dir/PyArray.cpp.o
In file included from /usr/include/python3.8/numpy/ndarraytypes.h:1830,
                 from ../../../pdal/PyArray.hpp:37,
                 from ../../../pdal/PyArray.cpp:35:
/usr/include/python3.8/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
[17/18] Install the project...
-- Install configuration: "Release"
-- Installing: /mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdal_plugin_reader_numpy.so
-- Set runtime path of "/mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdal_plugin_reader_numpy.so" to ""
-- Installing: /mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdal_plugin_filter_python.so
-- Set runtime path of "/mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdal_plugin_filter_python.so" to ""
-- Installing: /mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdalpython.cpython-38-x86_64-linux-gnu.so
-- Set runtime path of "/mnt/data/archives/python-2.3.6/_skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdalpython.cpython-38-x86_64-linux-gnu.so" to ""

copying pdal/array.py -> _skbuild/linux-x86_64-3.8/cmake-install/pdal/array.py
copying pdal/__init__.py -> _skbuild/linux-x86_64-3.8/cmake-install/pdal/__init__.py
copying pdal/dimension.py -> _skbuild/linux-x86_64-3.8/cmake-install/pdal/dimension.py
copying pdal/pio.py -> _skbuild/linux-x86_64-3.8/cmake-install/pdal/pio.py
copying pdal/pipeline.py -> _skbuild/linux-x86_64-3.8/cmake-install/pdal/pipeline.py

running build
running build_py
creating _skbuild/linux-x86_64-3.8/setuptools/lib
creating _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/array.py -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/dimension.py -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/pio.py -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/pipeline.py -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdal_plugin_reader_numpy.so -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdal_plugin_filter_python.so -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copying _skbuild/linux-x86_64-3.8/cmake-install/pdal/libpdalpython.cpython-38-x86_64-linux-gnu.so -> _skbuild/linux-x86_64-3.8/setuptools/lib/pdal
copied 5 files
running build_ext
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,5 +1,5 @@
 exclude MANIFEST.in
-include CHANGES.txt README.rst pyproject.toml tox.ini
+include CHANGES.txt README.rst pyproject.toml tox.ini CMakeLists.txt
 include pdal/*.txt
 include pdal/*.cmake
 include test/*

